tool/icons/update_icons.dart

```diff
@@ -159,6 +158,13 @@ Future<void> generateIcons(String appFolder) async {
       .transform(utf8.decoder)
       .transform(const LineSplitter())
       .listen((String line) {
+    // Uncomment following condition if you have custom PUB_HOSTED_URL set in your env.
+    // if (RegExp(
+    //   r'Flutter assets will be downloaded from https?://.+. '
+    //   r'Make sure you trust this source!',
+    // ).hasMatch(line)) {
+    //   return;
+    // }
```

Comment:
This helps when running locally with `PUB_HOSTED_URL` configured. For example, I've configured it as `https://storage.flutter-io.cn` which will raise a warning, and `stderr` treat it as an error line.